posts/urls.py

```diff
@@ -8,4 +9,5 @@
     url(r'^(?P<pk>\d+)/edit/$', views.PostEditView.as_view(), name='edit'),
     url(r'^(?P<pk>\d+)/repost/$', views.PostRepostView, name='repost'),
     url(r'^(?P<pk>\d+)/favorite/$', views.PostFavoriteView, name='favorite'),
+    url(r'^show/(?P<hashtag_name>\w+)/$', PostsListView.as_view(), name='posts_list'),
```

Comment:
This would correspond to a url like `localhost:8000/posts/show/foobar`?
I'd prefer something like `localhost:8000/posts/hashtag/foobar` even better would be `localhost:8000/posts?hashtag=foobar` or `localhost:8000/hashtag/foobar`.
The last one would not be in this `posts/urls.py` but in `circuit/urls.py`.

Which one do you prefer?